apps/dashboard/app/models/project.rb

```diff
@@ -145,11 +154,7 @@ def icon_class
 
   def destroy!
     remove_from_lookup
-    if directory.to_s.include?(Project.dataroot.to_s)
-      FileUtils.remove_dir(project_dataroot, force = true)
-    else
-      FileUtils.remove_dir(configuration_directory, force = true)
-    end
+    FileUtils.remove_dir(configuration_directory, force = true)
```

Comment:
IDK what your dev environment looks like - but you should use `solargraph` (I use the extension for VS Code).

It'll tell you simple mistakes like this: 

![image](https://github.com/OSC/ondemand/assets/4874123/3543bf76-9bf8-4cdc-b4b4-def6e0c4da7e)

Reply:
yea really it's just a rubocop rule, solargraph is just the language server.  Looks like jetbrains has support for it.

There's a `.rubocop.yml` at the top level of this project with a rake task to copy it to a more local directory (if you're like me and just open the `apps/dashboard` directory instead of the entire project).